extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -133,7 +114,11 @@ private function renderGroup( $groupData ) {
 			foreach ( $dataItems as $item ) {
 				$type = $item[ 'type' ];
 
-				if ( $helper->isTypeSupportedInTemplates( $type, self::getTemplates() ) ) {
+				if ( $this->templateEngine->isSupportedType( $type ) ) {
```

Comment:
It's doing exactly the same thing as https://github.com/Wikia/app/pull/12144/files#diff-38f4ac017c46ec2a05d85a08356fab19R40. Maybe we could extract this logic to a method?